app/models/supported_ssg.rb

```diff
@@ -9,6 +9,14 @@
   self::SUPPORTED_FILE = Rails.root.join('config/supported_ssg.yaml')
 
   class << self
+    def supported?(ssg_version:, os_version:)
```

Comment:
The OS version has to be constructed by the consumer, e.g. `"#{@host.os_major_version}.#{@host.os_minor_version}"`.
How about passing major and minor OS version with args and have the OS version, or specifically the key, constructed in this class?

Reply:
I'm fine with it either way tbh. The inventory returns major and minor together, but we split it into major and minor when we store it in our DB and obviously have to display it as X.Y most places it's displayed. IMO it'd be overkill at this point to write some service class that splits or joins version numbers. It's unclear to me why the SupportedSsg class would be the place to put this logic.

Reply:
Throughout the code base we should use `os_major_version` and `os_minor_version`. `SupportedSsg` constructs the key to access the data, that's it's own thing. The key can have any format, and might not necessarily match the joined OS version.

We do not necessarily have to create a separate service that would handle OS versions, at least not now.  I guess the project Cyndi and/or enhancements in Inventory could help with this concern.

Reply:
Updated to split into major/minor.

Reply:
Thanks :+1: 

You could have done the same all the way to `ssg_for_os` method (as it is public). :wink:

Reply:
`ssg_for_os` is now also split into major/minor